Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/LuceneIndexController.java

```diff
@@ -0,0 +1,110 @@
+package edu.asu.conceptpower.servlet.web;
+
+import java.security.Principal;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import edu.asu.conceptpower.servlet.core.ErrorConstants;
+import edu.asu.conceptpower.servlet.core.IIndexService;
+import edu.asu.conceptpower.servlet.core.IndexingEvent;
+import edu.asu.conceptpower.servlet.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.servlet.exceptions.LuceneException;
+import edu.asu.conceptpower.servlet.lucene.ILuceneDAO;
+
+/**
+ * This class provides methods for deleting and viewing lucene indexes
+ * 
+ * @author mkarthik90
+ *
+ */
+@Controller
+public class LuceneIndexController {
+
+    @Autowired
+    private IIndexService manager;
+
+    @Autowired
+    @Qualifier("luceneDAO")
+    private ILuceneDAO dao;
+
+    /**
+     * Retrives latest indexing time and number of concepts indexed so far
+     * 
+     * @param model
+     * @return
+     */
+    @RequestMapping(value = "auth/loadIndex", method = RequestMethod.GET)
```

Comment:
just call this "auth/index", load index makes it sound like it loads the index, but it really doesn't. It should shows you a page to reindex and with some info.